src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
Consider encapsulating the `fruits` map by providing methods to access and modify it. This will help prevent unintended modifications from outside the `Storage` class and maintain better control over the data.